apps/site/pages/en/download/current.mdx

```diff
@@ -27,10 +27,10 @@ Read the <Release.BlogPostLink>blog post</Release.BlogPostLink> for this version
 
 Learn how to <LinkWithArrow href="https://github.com/nodejs/node#verifying-binaries">verify</LinkWithArrow> signed SHASUMS.
 
-Learn how to <LinkWithArrow href="https://github.com/nodejs/node/blob/main/BUILDING.md#building-nodejs-on-supported-platforms">build Node.js</LinkWithArrow> from source.
+Looking for Node.js source? Download a signed <Release.DownloadLink kind="source">Node.js source</Release.DownloadLink> tarball.
 
 Check out our <LinkWithArrow href="https://nodejs.org/download/nightly/">nightly</LinkWithArrow> binaries or
-all <Link href="/about/previous-releases#looking-for-the-latest-release-of-a-version-branch">previous releases</Link>
+all <LinkWithArrow href="https://nodejs.org/download/release/">previous releases</LinkWithArrow>
```

Comment:
nit (not sur eif it matters)
```suggestion
all <LinkWithArrow href="/download/release/">previous releases</LinkWithArrow>
```

Reply:
In this case it needs to have https:// otherwise Next.js will treat as an internal link and attempt to prefetch it.